doc/nrf/external_comp/memfault.rst

```diff
@@ -110,25 +116,26 @@ Add the following code in the :file:`CMakeLists.txt` file to have these configur
 
 See the `Memfault nRF Connect SDK integration guide`_ for more details.
 
-.. _ug_memfault_error:
+.. note::
+   In the |NCS| samples and applications folders that use Memfault, these configuration files are already added.
 
 Error Tracking
-==============
+**************
```

Comment:
Line 122: Perhaps "Error tracking"?

Reply:
In the Memfault docs, it is mentioned as "Error Tracking" - https://docs.memfault.com/docs/mcu/trace-events. So, better to keep it as is.

Reply:
This needs to be checked.

Reply:
Checked with Jamie and Error tracking should be used. Will update the docs, wherever applicable.